### DISCUSSION TITLE
"/" not working

### DISCUSSION THREAD

**Discussion by qwertyaos** [psoukie/zipchord#136] [Q&A] [answered]
The best way to submit feedback and ask questions is through [Discussions](https://github.com/psoukie/zipchord/discussions), but you can also submit your question or feedback using this form by writing your comments here:

Can / key work for chord or shorthand? I added it to the dictionary to use it like w/o for without but not working.

**Comment by psoukie** [comment, marked as answer]
By default, ZipChord considers `/` as punctuation (and punctuation can end a shorthand).

To change this, click **Customize** under Keyboard and language, delete `/` from where it appears under "Unmodified keys" (but not where it is under "If Shift was pressed"). Then click Save Changes, Close, and Apply/OK, and this should work. (I just tried this on my computer.)

Please let me know if it doesn't work.

**Reply by qwertyaos** [reply]
Perfectly working with your tip!
